linchpin/shell/__init__.py

```diff
@@ -466,10 +482,22 @@ def journal(ctx, targets, fields, count, view, tx_id):
                 output += '\n==================NO TRANSACTIONS======'
                 output += '==========\n'
 
-
-        # PRINT OUTPUT RESULTS HERE
         ctx.log_state(output)
 
+    if view == "inventory":
```

Comment:
As I stated above, I don't think 'inventory' is a view type, but an output type. We look at transactions, (or target/run_id combination) as views. Maybe allow output-format or some such to handle the details. Here's some code that may describe it better: 

```
if output_format == 'json-inventory':
    .. snip ..
elif output_format == 'yaml-inventory':
    .. snip ..
elif output_format == 'cfg-inventory':
   .. snip ..
```
And add other options here. Clearly this is probably not the best way to use the inventory format, but I think it gets the point across.